packages/dds/merge-tree/src/test/obliterate.rangeExpansion.spec.ts

```diff
@@ -357,37 +357,26 @@ describe.skip("reconnect", () => {
 		action: (helper) => {
 			helper.insertText("A", 0, "hello world");
 			helper.processAllOps();
-			helper.disconnect(["C"]);
-			const op = helper.obliterateRangeLocal(
-				"C",
-				{ pos: 1, side: Side.After },
-				{ pos: 4, side: Side.After },
-			);
-			helper.reconnect(["C"]);
-			helper.submitDisconnectedOp("C", op);
-			helper.processAllOps();
+			helper.disconnect("C");
```

Comment:
these tests are skipped but I was prototyping these changes on a branch that should get obliterate+reconnect working. With the fixes I made here, they do pass on that branch.

Specific justification for the fixes:

1. The first test needs to submit the insertion of "123" on "A" before A sees the obliterate. Otherwise the expectation would be that the 123 segment actually exists. The updated lines seem to reflect the intent of the test based on naming and are the more interesting case to test.
2. Both test cases obliterate from after 1 to after 4, so 3 characters. That means the "llo" in "hello" is gone. :)